dev-support/benchmark/grouping/grouping_conf

```diff
@@ -0,0 +1,16 @@
+#Grouping Benchmark
+prefix:/usr/local/flink-1.0.0/bin/flink
+class:org.gradoop.examples.GroupingBenchmark
+jar:gradoop-examples.jar
+parallelism: 16
+rounds: 3
+input: /user/hduser/input/1_json_merged/
+output: /user/hduser/output
+csv: /home/hduser/csv/grouping_csv
+vgk: city,age,university
+egk: since
+vagg: max
+vak: since
+vark: count(since)
```

Comment:
Thats right ít should be max(since).

Aggregator functions, aggregator keys and result keys must be defined in order.

for example:

agg functions:    max, max, min
agg keys            income, since, age
agg result keys  max(income), max(since), min(age)